### PR TITLE
Make bearer token not required in jira plugin

### PR DESCRIPTION
**Pull request by Irma12**
Currently we suppose users will be using Basic auth in order to authenticate. However, some users are using Bearer token so we need to take that into consideration.

### CODE REVIEW

**Review by Xantier — APPROVED** [review]
Could use a README.md change also.

**Comment by Irma12** [comment]
@Xantier Agree, will add it

**Comment by martina-if** [comment]
fixes #277

File: .changeset/real-steaks-destroy.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/backstage-plugin-jira': minor
+---
+
+Provided an option to use Bearer token fo retrieve activity stream.
```
Comment (martina-if) [inline .changeset/real-steaks-destroy.md:5]: small typo `fo` -> `to`

**Review by Xantier — APPROVED** [review]

**Pull request merged**